packages/shylu/shylu_dd/bddc/src/shylu_DofManager.h

```diff
@@ -54,18 +54,19 @@ using Teuchos::rcp;
 
 #include "Tpetra_Map.hpp"
 #include "Tpetra_CrsMatrix.hpp"
+#include "shylu_errorBDDC.h"
 
 // Author: Clark R. Dohrmann
 namespace bddc {
-
-template <class LO, class GO>
+  
+template <class LO, class GO> 
   class DofManager
 {
 public:
   //
   // Convenience typedefs
   //
-  typedef Tpetra::Map<>::node_type  Node;
+  typedef Tpetra::Map<>::node_type                                Node;
   typedef Tpetra::Map<LO,GO,Node>                                 Map;
```

Comment:
If you're only ever using the default Node type, then don't specify the Node type.  For example:
```c++
typedef Tpetra::Map<LO,GO> Map;
```

Reply:
Done (in this file and others).